docs/src/gettingstarted/installation.md

```diff
@@ -3,9 +3,38 @@
 
 ## Short Version
 
-From the Julia REPL, run: 
+First, you will need to install julia (see below for instructions). 
+
+Create a folder to hold your julia `ACE` project, and `cd` into the fodler. This folder will track the packages and versions which the `ACE1pack` code requires. For example:
+
+```
+mkdir ~/ACE1project
+cd ~/ACE1project
+```
+
+from within this folder type julia to enter the Julia REPL. Then run
 ```julia
-using Pkg; Pkg.activate("."); pkg"registry add https://github.com/JuliaRegistries/General"; pkg"registry add https://github.com/JuliaMolSim/MolSim.git"; pkg"add ACE1pack"
+using Pkg; Pkg.activate("."); pkg"registry add https://github.com/JuliaRegistries/General"; pkg"registry add https://github.com/JuliaMolSim/MolSim.git"; pkg"add ACE1pack, ACE1, JulIP, IPFitting, ASE"
+```
+
+from now on, you will need to export the environment variable `JULIA_PROJECT` set to the path to this folder. For example, `export JULIA_PROJECT=~/ACE1project`. You can add this to your `.bashrc`/`.bash_profile` and not touch it again. 
```

Comment:
I think the sentence "You can add this to your `.bashrc`/`.bash_profile` and not touch it again. " needs to be removed. If somebody has 2 or more projects then this will be very confusing.